src/main.ts

```diff
@@ -35,7 +35,12 @@ const run = async () => {
         inlineScript = ` set -e >&2; echo '${START_SCRIPT_EXECUTION_MARKER}' >&2; ${inlineScript}`;
         scriptFileName = await createScriptFile(inlineScript);
         let startCommand: string = ` ${BASH_ARG}${CONTAINER_TEMP_DIRECTORY}/${scriptFileName} `;
-
+        let gitHubEnvironmentVariables = '';
+        for (let key in process.env){
+            if (key.toUpperCase().startsWith("GITHUB_") && key.toUpperCase() !== 'GITHUB_WORKSPACE' && process.env[key]){
+                gitHubEnvironmentVariables += ` -e ${key}=${process.env[key]} `;

```

Comment:
a space both and start and end of each variable looks redundant

Reply:
yeah added it so as to avoid any unforeseen issues.

Reply:
okay